### ISSUE TITLE
Should I apply only to the discriminator?

### ISSUE THREAD

**Issue by taki0112** [christiancosgrove/pytorch-spectral-normalization-gan#4]
Should I apply only to the discriminator?

Do not apply spectral normalization to both generator and discriminator?

**Comment by christiancosgrove** [comment]
In the paper, only the discriminator is trained with spectral normalization. The generator uses batch norm.

"We trained the parameters of the
generator with batch normalization" (p. 5 [Miyato et al.](https://openreview.net/pdf?id=B1QRgziT-))
